File: PYTHON/tpl-0004.py
#!/usr/bin/env python

# --- TEXTPATGEN-QUICKSCRIPT-PYTHON ---

# Users can change the output by editing
# this file directly.

# The text is written to a timestamped file.

import time

fp=open(time.strftime('00_%Y%m%d_%H%M%S.txt'), 'w')
fp.write(time.strftime('TEXTPATGEN-QUICKSCRIPT-PYTHON: %Y-%m-%dT%H:%M:%S%z\n'))
num=0
for length in range(0, 25):
  for width in range(0, 19):
    fp.write('+')
    num=num+1
    width=width+1
    length=length+1
  fp.write('+\n')
  num=num+1
fp.close()

File: LUA/tpl-0004.lua
-- TEXTPATGEN-QUICKSCRIPT-LUA
-- Please edit this file to suit your preferences.
-- The text is written to a timestamped file.

filename = os.date("00_%Y%m%d_%H%M%S.txt")
file = io.open(filename, "w")
introstr = os.date("TEXTPATGEN-QUICKSCRIPT-LUA: %Y-%m-%dT%H:%M:%S%z\n")
file:write(introstr)
n3 = 0
for n1 = 0, 15 do
  for n2 = 0, 14 do
    str = string.format("+", n3)
    file:write(str)
    n3 = n3 + 1
    end 
  str = string.format("+\n", n3)
  file:write(str)
  n3 = n3 + 1
end
file:close()
